Assets/AnkrSDK/Scripts/Core/Utils/EventFilterHelper.cs

```diff
@@ -6,35 +6,18 @@ namespace AnkrSDK.Core.Utils
 {
 	public static class EventFilterHelper
 	{
-		public static NewFilterInput CreateEventFilters(EventBase eventHandler, EventFilterData evFilter = null)
+		public static NewFilterInput CreateEventFilters<TEvDto>(string contractAddress, EventFilterData evFilter = null)
 		{
-			if (evFilter == null)
-			{
-				return eventHandler.CreateFilterInput();
-			}
-
-			var isFilterTopic1Filled = evFilter.filterTopic1 != null;
-			var isFilterTopic2Filled = evFilter.filterTopic2 != null;
-			var isFilterTopic3Filled = evFilter.filterTopic3 != null;
+			var eventABI = ABITypedRegistry.GetEvent(typeof(TEvDto));
```

Comment:
use .GetEvent<TEvDto>()

Reply:
Also you should check if eventABI is not null. If it is null we should log an error

Reply:
> Also you should check if eventABI is not null. If it is null we should log an error
it alredy done inside GetEvent 
<img width="723" alt="image" src="https://user-images.githubusercontent.com/38144180/163757289-f5c59db7-a1fe-48b2-bf8a-5d8ffcc2fb8b.png">

Reply:
> use .GetEvent()

What .GetEvent() do you mean?

Reply:
Generic .GetEvent<>()